modules/porous_flow/src/actions/PorousFlowAddMaterialJoiner.C

```diff
@@ -28,7 +28,7 @@ validParams<PorousFlowAddMaterialJoiner>()
 }
 
 PorousFlowAddMaterialJoiner::PorousFlowAddMaterialJoiner(const InputParameters & params)
-  : Action(params)
+  : Action(params), _already_joined()
```

Comment:
You probably know this, but you don't need to explicitly initialize container types.